apps/aehttp/src/aehttp_app.erl

```diff
@@ -96,10 +96,13 @@ local_peer(Port) ->
                     erlang:error({cannot_parse, [{local_peer_address,
                                                   Addr}]});
                 nomatch ->
-                    aec_peers:uri_from_ip_port(Addr, Port)
+                    "http://" ++ Addr ++ ":" ++ integer_to_list(Port) ++ "/"
             end
     end.
 
+%%------------------------------------------------------------------------------
+%% Get url from IP and port. IP format: xxx.xxx.xxx.xxx
+%%------------------------------------------------------------------------------
 
```

Comment:
The comment suggests that you intended to define the `uri_from_ip_port` function locally and use it in the cases above. That would be nicer than inlining it.